src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -421,4 +429,28 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
         }
     }
 
+    /** Check whether the overlay or button has focus */
```

Comment:
This now only checks the overlay, so the comment (and probably function name) needs updating.